src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -542,6 +542,9 @@ private SupportedPlatform GetCurrentRunningOS()
                     DefaultDriverOptions.CreateRuleSpecificOption(skimmer, DefaultDriverOptions.RuleEnabled);
 
                 RuleEnabledState ruleEnabled = rootContext.Policy.GetProperty(ruleEnabledProperty);
+                FailureLevel failureLevel = (ruleEnabled == RuleEnabledState.Default || ruleEnabled == RuleEnabledState.Disabled)
```

Comment:
>failureLevel [](start = 29, length = 12)

generate failureLevel from the file. With that, we can replace our default value

Reply:
ok! but we need more unit tests here, clearly. :) maybe yong can help?

---
In reply to: [541631128](https://github.com/microsoft/sarif-sdk/pull/2181#discussion_r541631128) [](ancestors = 541631128)